hoomd/AABB.h

```diff
@@ -324,6 +335,7 @@ DEVICE inline bool contains(const AABB& a, const AABB& b)
     }
 
 
+#ifndef __HIPCC__
```

Comment:
We could use a platform agnostic `min()` so that this method is available in both code paths.

Reply:
for now I'd leave it as is, as `merge` is usually called in the context of leaf node updates, which require a serial code path (or atomics) to avoid race conflicts. If we need to use `merge` on the GPU in the future, we can make that modification